errors/NoInstanceFound.md

```diff
@@ -36,3 +36,66 @@ logGet param = do
 will throw this error. This is because the type signature is not properly constructed, and should account for the `CONSOLE` effect arising from the `log` function. The type checker will not infer that `CONSOLE` should be a part of this type signature in its warning if the type signature is left out, *probably* due to the lifting of the effect. The correct type signature is:
 
 ``` logGet :: forall aff. String -> Aff (ajax :: AJAX, console :: CONSOLE | aff) Unit```
+
+
+### exhaustivity
```

Comment:
Correct: the compiler now generates Partial constraints on inexhaustive pattern matches, and so if a function is declared as not being partial but uses a nonexhaustive pattern match, this becomes a NoInstanceFound error as the compiler will try and fail to find a Partial instance.